config/ad_config.py

```diff
@@ -0,0 +1,616 @@
+    STRINGS = {
+        "demultiplexlog_tso500_msg": DEMULTIPLEXLOG_TSO500_MSG,
+        "lane_metrics_suffix": LANE_METRICS_SUFFIX,
+        "demultiplex_success": DEMULTIPLEX_SUCCESS,
+    }
+    UPLOAD_STARTED_MSG = (
+        "Upload started"  # Statement to write to DNAnexus upload started file
+    )
+    # TODO move this to the DXAPP.JSON file for FH app
+    PIPE_FH_GATK_TIMEOUT_ARGS = (
+        # Set 6 hour timeout policy for gatk app and jobtimeoutexceeded
+        # reason to auto restart list
+        '--extra-args \'{"timeoutPolicyByExecutable": {"'
+        f'{NEXUS_IDS["APPS"]["gatk"].split(":")[1]}'
+        '": {"*":{"hours": 12}}}, "executionPolicy": {"restartOn": {"JobTimeoutExceeded":1, "JMInternalError":'
+        ' 1, "UnresponsiveWorker": 2, "ExecutionError":1}}}\''
+    )
+    QUERIES = {
+        # SQL queries for updating the Moka database (audit trail)
+        "customrun": "insert into NGSCustomRuns(DNAnumber,PipelineVersion,RunID) values (%s)",
+        "wes": "update NGSTest set PipelineVersion = %s, StatusID = %s where dna in ('%s') and StatusID = %s",
+        "oncology": "insert into NGSOncologyAudit(SampleID1,SampleID2,RunID,PipelineVersion,ngspanelid) values (%s)",
+    }
+    PSCON_IDS = [
+        "NA12878",
+        "136819",  # NA12878
+        "HD200",  # Seracare v4 tumour fusion reference material
+    ]
+    NTCON_IDS = [
+        "00000",
+        "NTCcon",
+        "NTC000",
+        "NC000"
+    ]
+    SQL_IDS = {
+        # Moka IDs for generating SQLs to update the Moka database (audit trail)
+        "WORKFLOWS": {
+            "pipe": 5229,
+            "wes": 5078,
+            "archerdx": 5238,
+            "snp": 5091,
+            "tso500": 5288,
+        },
+        "WES_TEST_STATUS": {
+            "nextseq_sequencing": 1202218804,  # Test Status = NextSEQ sequencing
+            "data_processing": 1202218805,  # Test Status = Data Processing
+        },
+    }
+    STAGE_INPUTS = {
+        "pipe": {
+            "fastqc_reads": f"-i{NEXUS_IDS['STAGES']['pipe']['fastqc']}.reads=",
+            "bwa_instance": f"--instance-type {NEXUS_IDS['STAGES']['pipe']['bwa']}=mem1_ssd1_v2_x8",
+            "bwa_reads1": f"-i{NEXUS_IDS['STAGES']['pipe']['bwa']}.reads_fastqgz=",
+            "bwa_reads2": f"-i{NEXUS_IDS['STAGES']['pipe']['bwa']}.reads2_fastqgz=",
+            "bwa_rg_sample": f"-i{NEXUS_IDS['STAGES']['pipe']['bwa']}.read_group_sample=",
+            "bwa_ref": f"-i{NEXUS_IDS['STAGES']['pipe']['bwa']}.genomeindex_targz=",
+            "picard_instance": f"--instance-type {NEXUS_IDS['STAGES']['pipe']['picard']}=mem1_ssd1_v2_x4",
+            "picard_bed": f"-i{NEXUS_IDS['STAGES']['pipe']['picard']}.vendor_exome_bedfile=",
+            "picard_capturetype": f"-i{NEXUS_IDS['STAGES']['pipe']['picard']}.Capture_panel=",
+            "gatk_instance": f"--instance-type {NEXUS_IDS['STAGES']['pipe']['gatk']}=",
```

Comment:
Sorry I don't understand the comment

Reply:
i think I was querying why the instance isn't actually specified, but I realise that it's because a different instance is specified for GATK depending on whether it's an FH sample or not